src/nft/components/explore/Explore.css.ts

```diff
@@ -106,3 +106,60 @@ export const carouselIndicator = sprinkles({
   borderRadius: 'round',
   display: 'inline-block',
 })
+
+/* Value Prop Styles */
+export const valuePropWrap = sprinkles({
+  width: 'full',
+  borderWidth: '1px',
+  borderStyle: 'solid',
+  borderColor: 'grey700',
+  borderRadius: '12',
+  paddingLeft: '16',
+  paddingRight: '16',
+  marginTop: '60',
+  position: 'relative',
+})
+
+export const valuePropOverlay = style([
+  {
+    height: '135px',
+  },
+  sprinkles({
+    position: 'absolute',
+    zIndex: '0',
+    width: 'full',
+    backgroundColor: 'grey900',
+    left: '0',
+    top: '0',
+    opacity: { sm: '0.7', xl: '0.1' },
+  }),
+])
+
+export const valuePropContent = style([
+  sprinkles({
+    position: 'relative',
+    zIndex: '1',
+    paddingLeft: '20',
+    paddingBottom: '18',
+    fontSize: '20',
+    paddingTop: '28',
+  }),
+  {
+    lineHeight: '28px',
+    '@media': {
+      [`(max-width: 400px)`]: { width: '88%' },
+      [`(min-width: 400px)`]: { width: '67%' },
+      [`(min-width: ${breakpoints.sm}px)`]: {
+        width: '74%',
+        paddingLeft: '36px',
+        fontSize: '28px',
+        lineHeight: '36px',
+        paddingTop: '32px',
+      },
+      // Extra breakpoint for text wrapping
+      [`(min-width: 800px)`]: { width: '60%' },
+      [`(min-width: ${breakpoints.md}px)`]: { width: '58%' },
+      [`(min-width: ${breakpoints.lg}px)`]: { width: '50%' },
```

Comment:
sprinkle

Reply:
how so? We can't specify width as a percentage with sprinkles (besides `half` and `full`), can we?

Reply:
specifically for the 50% width breakpoint we can use `half`

Reply:
Moving that to a sprinkle isn't working - the width gets overridden by the media query. Gonna leave as is.